### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi 40e735e.
r? @brandur-stripe
cc @stripe/api-libraries

## Changelog
* Added support for `subscription_pause` on `Configuration#update.features`, `Configuration#create.features` and `BillingPortal.Configuration.features`

### CODE REVIEW

**Comment by brandur-stripe** [comment]
Looks like a couple minor lint problems on this one:

```
billingportal_configuration.go:109:6: exported type BillingPortalConfigurationFeaturesSubscriptionPauseParams should have comment or be unexported
billingportal_configuration.go:202:6: exported type BillingPortalConfigurationFeaturesSubscriptionPause should have comment or be unexported
```

File: billingportal_configuration.go [hunk 1]
```diff
@@ -106,6 +106,9 @@ type BillingPortalConfigurationFeaturesSubscriptionCancelParams struct {
 	Mode              *string `form:"mode"`
 	ProrationBehavior *string `form:"proration_behavior"`
 }
+type BillingPortalConfigurationFeaturesSubscriptionPauseParams struct {
```
Comment (richardm-stripe) [inline billingportal_configuration.go:109]:
```suggestion
// BillingPortalConfigurationFeaturesSubscriptionPauseParams lets you pass details on the
// subscription pause on a portal configuration.
type BillingPortalConfigurationFeaturesSubscriptionPauseParams struct {
```

File: billingportal_configuration.go [hunk 2]
```diff
@@ -195,6 +199,9 @@ type BillingPortalConfigurationFeaturesSubscriptionCancel struct {
 	Mode              BillingPortalConfigurationFeaturesSubscriptionCancelMode              `json:"mode"`
 	ProrationBehavior BillingPortalConfigurationFeaturesSubscriptionCancelProrationBehavior `json:"proration_behavior"`
 }
+type BillingPortalConfigurationFeaturesSubscriptionPause struct {
```
Comment (richardm-stripe) [inline billingportal_configuration.go:202]:
```suggestion
// BillingPortalConfigurationFeaturesSubscriptionPause lets you pass pause details
// on the subscription update on a portal configuration.
type BillingPortalConfigurationFeaturesSubscriptionPause struct {
```

**Comment by richardm-stripe** [comment]
r? @brandur all fixed!

Sorry for the premature ping, you're tagged as the "primary_reviewer" for go in the bindings repo so it automatically tagged you.

**Review by brandur-stripe — APPROVED** [review]
> Sorry for the premature ping, you're tagged as the "primary_reviewer" for go in the bindings repo so it automatically tagged you.

Ah! No worries.

**Pull request merged**